component.yml

```diff
@@ -0,0 +1,8 @@
+component:
+  sbom_info:
+    - name: grpc
+      version: 1.69.0
+      identifiers:
+        cpe: cpe:2.3:a:grpc:grpc:1.69.0:*:*:*:*:-:*:*
+      licenses:
+        - legal_package: gRPC151
```

Comment:
In our installers I see some `gRPC157` instances but that's the highest I see. You should ask Fred if that is sufficient or if a newer one is warranted.